### PR TITLE
Refactor createVariableDeclarationStatement methods

### PR DESCRIPTION
**Pull request by davidmitesh**
More details on the refactor can be found in [this](https://www.notion.so/nethermind/Warp-2-0-Transpiler-af80a911dd8742b28604aa604fe6455f?p=3bc8f5d11b3644eb9a14d77bd9d91fe4&pm=s) ticket on the job board.

### CODE REVIEW

File: src/passes/expressionSplitter/expressionSplitter.ts [hunk 1]
```diff
@@ -65,13 +69,32 @@ export class ExpressionSplitter extends ASTMapper {
...
+        initialValue.typeString,
       );
+      ast.setContextRecursive(varDecl);
+
+      // const tempVarStatement = createVariableDeclarationStatement(
+      //   this.eGen.next().value,//name
+      //   node.vRightHandSide,//initialValue
+      //   ast.getContainingScope(node),//scope
+      //   ast,//ast
+      // );
```
Comment (cicr99) [inline src/passes/expressionSplitter/expressionSplitter.ts:96]: These comments can be removed

File: src/passes/expressionSplitter/expressionSplitter.ts [hunk 2]
```diff
@@ -173,38 +196,38 @@ function identifierReferenceStateVar(id: Identifier) {
...
+
+//   const varDeclStatement = new VariableDeclarationStatement(
+//     ast.reserveId(),
+//     '',
+//     [varDecl.id],
+//     [varDecl],
+//     cloneASTNode(initialValue, ast),
+//   );
+//   return varDeclStatement;
+// }
```
Comment (cicr99) [inline src/passes/expressionSplitter/expressionSplitter.ts:230]: These comments can be removed as well

File: src/passes/expressionSplitter/expressionSplitter.ts (hunk 1 above)
Comment (davidmitesh) [reply src/passes/expressionSplitter/expressionSplitter.ts]: done

File: src/passes/expressionSplitter/expressionSplitter.ts (hunk 2 above)
Comment (davidmitesh) [reply src/passes/expressionSplitter/expressionSplitter.ts]: done

**Review by cicr99 — APPROVED** [review]

**Review by RohitRanjangit — APPROVED** [review]

**Comment by GuyMcComb** [comment]
Sem Tests pass

**Pull request merged**